sycl/test-e2e/KernelFusion/abort_fusion.cpp

```diff
@@ -1,5 +1,5 @@
 // RUN: %{build} %{embed-ir} -o %t.out
-// RUN: env SYCL_RT_WARNING_LEVEL=1 %{run} %t.out 2>&1 | FileCheck %s
+// RUN: env SYCL_RT_WARNING_LEVEL=1 %{run} %t.out 2>&1 FileCheck %s
```

Comment:
I think most of these changes are unneeded. Can we revert the unneeded changes in kernel fusion tests?

Reply:
That was a wrong decision on making the PR ready for review, this was only for testing purposes.